python/hail/tests/tests_v0_1_cf0e0137a.py

```diff
@@ -860,11 +863,12 @@ def test_concordance(self):
     def test_hadoop_methods(self):
         data = ['foo', 'bar', 'baz']
         data.extend(map(str, range(100)))
+        data = [bytearray(d, encoding="utf8") for d in data]
```

Comment:
in which case the 'r' and 'w' default to 'rt' and 'wt' and you need to explicitly ask for binary with 'rb' and 'wb'